src/core/components/Search/SearchResults.js

```diff
@@ -31,8 +31,10 @@ export default class SearchResults extends React.Component {
 
     let searchResults;
     let messageText;
+    let hideMessageText = false;
 
     if (query && count > 0) {
+      hideMessageText = true;
```

Comment:
I did this so the search result count would be there for screenreaders, which I thought might be nice.

Maybe this isn't worthwhile though and it's best to just remove.